src/main/java/mat/model/clause/ModelTypeHelper.java

```diff
@@ -0,0 +1,30 @@
+package mat.model.clause;
+
+public final class ModelTypeHelper {
```

Comment:
Would there be any benefit to converting this class to an Enum?

Reply:
I would gladly convert it into an enum. But there are some hibernate mapping concerns and that's a lot of changes on server-side and UI end. So, I'm a bit hesitant about that.

Reply:
Fair enough, I was thinking there might be a large testing effort behind the idea.